flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/source/reader/MySqlSplitReader.java

```diff
@@ -74,6 +78,7 @@ public MySqlSplitReader(Configuration config, int subtaskId) {
             LOG.warn("fetch data failed.", e);
             throw new IOException(e);
         }
+        metricRecorder.recordProcessTime(System.currentTimeMillis());
```

Comment:
The process time means the processing time of Source Operator, thus I think we can also record it in `MySqlRecordEmitter` and update it per record. This will make the SplitReader clean.